tests/unit/test_apigateway.py

```diff
@@ -513,3 +515,36 @@ def test_construct_invocation_event(self):
                 t["is_base64_encoded"],
             )
             assert result == t["expected"]
+
+
+class TestRequestParameterResolver:
+    def test_resolve_request_parameters(self):
+        integration: Dict[str, Any] = {
+            "requestParameters": {
+                "integration.request.path.pathParam": "method.request.path.id",
+                "integration.request.querystring.baz": "method.request.querystring.baz",
+                "integration.request.querystring.token": "method.request.header.Authorization",
+                "integration.request.querystring.env": "stageVariables.enviroment",
+                "integration.request.header.Content-Type": "'application/json'",
```

Comment:
Are the double quotes intentional `"'` ?

Reply:
Yep, part of the spec